components/radio/package.json

```diff
@@ -1,24 +1,16 @@
 {
-  "//": [
-    "================================================================================",
-    "# To work within the development environment, run the following tasks",
-    "  1. $ npm run dev",
-    "  2. Go to http://localhost:8000",
-    "================================================================================"
-  ],
-  "name": "@aurodesignsystem/auro-radio",
+  "name": "@auro-formkit/auro-radio",
   "version": "4.1.1",
   "description": "auro-radio HTML custom element",
+  "type": "module",
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "license": "Apache-2.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/AlaskaAirlines/auro-radio"
   },
-  "type": "module",
-  "main": "index.js",
-  "license": "Apache-2.0",
-  "engines": {
-    "node": "^18 || ^20"
-  },
   "dependencies": {
     "@aurodesignsystem/auro-formvalidation": "^1.0.4",
     "@aurodesignsystem/auro-library": "~2.8.0",
```

Comment:
@aurodesignsystem/auro-library version here is also not aligned

Reply:
Moved `auro-library` dependency to root `package.json` and removed from component packages.